deploy/complete/helm-chart/mushop/charts/nats/values.yaml

```diff
@@ -5,6 +5,11 @@ image:
   tag: 2.1.2
   pullPolicy: IfNotPresent
 
+exporter: 
+  repository: synadia/prometheus-nats-exporter
```

Comment:
Is this the preferred way/image to export nats metrics?

Reply:
Yes it is. [Docs](https://github.com/nats-io/prometheus-nats-exporter#run) use the same image as well.
Synadia is the "commercial" face of the folks who build NATS.io. 
It simply runs the `prometheus-nats-exporter`. 

We could replace this with our own `Dockerfile` or use the [bitnami image](https://hub.docker.com/r/bitnami/nats-exporter) which is Apache 2.0 licensed.